src/customGraphQLValidationRules.js

```diff
@@ -111,22 +113,26 @@ export function RequiredFields(context, options) {
     // enough to have some child fragment to include the field, since we don't
     // know if that fragment covers all of the possible type options.
     Field(node) {
-      const def = context.getFieldDef();
-      if (!def) {
+      const type = context.getType();
```

Comment:
I'm not sure I can do this, but I've seen that it's the way it's done in other visitors :-)